hull3/gc_functions.sqf

```diff
@@ -79,8 +79,10 @@ hull3_gc_fnc_adjustConfig = {
 
 hull3_gc_fnc_sortDead = {
     params ["_killed"];
+
+    if (isNull _killed) exitWith {};
+    if (!(_killed isKindOf "CAManBase") || { !(_killed isKindOf "Car") } || { !(_killed isKindOf "Tank") } || { !(_killed isKindOf "Air") } || { !(_killed isKindOf "Ship") } ) exitWith {};
```

Comment:
You have to replace the ORs with ANDs. Also the line is too long and should be split into multiple lines like this:
```
if (condition1
    && {condition2}
    && {condition3}) {
```